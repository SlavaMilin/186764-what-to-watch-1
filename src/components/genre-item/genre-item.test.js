import React from 'react';
import renderer from 'react-test-renderer';
import GenreItem from "./genre-item";

it(`render genre item`, () => {
  const tree = renderer.create(<GenreItem genre={`drama`}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
