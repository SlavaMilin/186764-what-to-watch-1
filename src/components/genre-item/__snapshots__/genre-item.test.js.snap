// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render genre item 1`] = `
<li
  className="catalog__genres-item"
>
  <a
    className="catalog__genres-link"
    href="#"
  >
    drama
  </a>
</li>
`;
